scripts/submit_skimsUL2016APV_mib.py

```diff
@@ -237,11 +233,11 @@
     "VBFHToTauTau_M125"                   : "-n 50    -x 3.766    -y 0.0632 --isDYlike",
 
     "ttHToNonbb_M125"                 : "-n 50  -x 0.5071   -y 0.423    --ttHToNonBB True  --isTTlike",
-    "ttHTobb_M125"                    : "-n 50  -x 0.5071   -y 0.577  ",
+    "ttHTobb_M125"                    : "-n 50  -x 0.5071   -y 0.577   --isTTlike", # but this will be most likely signal like, still fine to put hyp here
     "ttHToTauTau_M125"                : "-n 50  -x 0.5071   -y 0.0632  --isTTlike",
 
-    "ZH_HToBB_ZToLL_M-125"             : "-n 50    -x 0.880    -y 0.058816576 ",
-    "ZH_HToBB_ZToQQ_M-125"             : "-n 50    -x 0.880    -y 0.407161664",
+    "ZH_HToBB_ZToLL_M-125"             : "-n 50    -x 0.880    -y 0.058816576 --isDYlike",
+    "ZH_HToBB_ZToQQ_M-125"             : "-n 50    -x 0.880    -y 0.407161664 --isDYlike",
```

Comment:
does this (and other similar occurrences in this file) also require updating in 2016/17/18 submission scripts?

Reply:
you are right, updated !